src/V3EmitCFunc.h

```diff
@@ -241,11 +262,23 @@ class EmitCFunc VL_NOT_FINAL : public EmitCConstInit {
                     exprp = stmtp;
                 }
                 if (AstCNew* const newRefp = VN_CAST(exprp, CNew)) {
+                    if (classp && constructorNeedsProcess(classp)) {
+                        puts("(vlProcess, vlSymsp");
+                    } else {
+                        puts("(vlSymsp");
+                    }
+                    baseCtorCall = true;
                     putCommaIterateNext(newRefp->argsp(), true);
+                    puts(")");
                     break;
                 }
             }
-            puts(")");
+            if (!baseCtorCall) {
+                if (classp && constructorNeedsProcess(classp))
+                    puts("(vlProcess, vlSymsp)");
+                else
+                    puts("(vlSymsp)");
```

Comment:
More logical I think to always print "(", conditionally print "vlProcess, ", always print "vlSymsp)".

Reply:
Done